src/preview/font_match.rs

```diff
@@ -271,7 +262,7 @@ fn filter_fonts_with_preview_font_settings(
     specified_preview_font_names: &[String],
 ) -> Vec<FontDescriptor> {
     if specified_preview_font_names.is_empty() {
-        all_available_fonts.iter().cloned().collect()
+        all_available_fonts.to_vec()
     } else {
```

Comment:
Can't really recall what I was thinking, I think this should be as simple as:
```
 if specified_preview_font_names.is_empty() {
    all_available_fonts
} else { ... }
```

Reply:
b5dbb4e